webservices/resources/committees.py

```diff
@@ -246,14 +246,8 @@ def get_committee(self, args, page_num, per_page, committee_id, candidate_id):
                 else:
                     committees = committees.filter(getattr(Committee, argname)==args[argname])
 
-        # default year filtering
-        if args.get('year') is None:
-            earliest_year = int(sorted(default_year().split(','))[0])
-            # still going or expired after the earliest year we are looking for
-            committees = committees.filter(or_(extract('year', CommitteeDetail.expire_date) >= earliest_year, CommitteeDetail.expire_date == None))
-
         # Should this handle a list of years to make it consistent with /candidate ?
-        elif args.get('year') and args['year'] != '*':
+        if args.get('year') and args['year'] != '*':
```

Comment:
Ok, that makes sense. I'm fine with leaving it in, then, but I'd recommend removing the comment/question that's above that and replacing it with something to the effect of wanting to support '*' across all endpoints.